.github/workflows/deleted-files.yml

```diff
@@ -0,0 +1,38 @@
+name: Deleted files
+
+on:
+  pull_request:
+    branches:
+      master
+
+jobs:
+  list-deleted-files:
+    runs-on: ubuntu-latest
+    name: Detect deleted files
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Get changed docs md files
+        id: changed-files
+        uses: tj-actions/changed-files@v37
+        with:
+          files: docs/**/*.md
+      - name: Format list of deleted files
+        id: format-output
+        env:
+          ORIGINAL: '${{ steps.changed-files.outputs.deleted_files }}'
```

Comment:
As of now, the workflow is only reactive to **deleted** files. But it's actually more frequent for us to **rename** files, and there should also be a reminder for those situations.

I already checked and the GitHub Action also seems to output a separate list with the renamed files, so it should be a matter of perhaps appending the two lists together, or even outputting the two lists separately on the comment. Do you care to have a go at it? :nerd_face: